README.md

```diff
@@ -195,52 +195,78 @@ Install custom reporters as a development dependency and you can use the `--repo
 nyc report --reporter=<custom-reporter-name>
 ```
 
-## Excluding files
+## Selecting files for coverage
 
-You can tell nyc to exclude specific files and directories by adding
-an `nyc.exclude` array to your `package.json`. Each element of
-the array is a glob pattern indicating which paths should be omitted.
+By default, nyc only collects coverage for source files that are visited
+during a test.  It does this by watching for files that are `require()`'d 
+during the test.  Only source files that are visited during a test will appear 
+in the coverage report and contribute to coverage statistics.  
+
+Nyc will cover all files if the `--all` flag is set.  In this case all files will
```

Comment:
nyc will instrument all files...

Reply:
I toyed with instrument there, but thought it may be a little too specific.  You're right though, only tests can cover source.